tests/SprykerEcoTest/Zed/AfterPay/Business/AfterPayPreCheckPluginTest.php

```diff
@@ -0,0 +1,170 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEcoTest\Zed\AfterPay\Business;
+
+use Generated\Shared\Transfer\ItemTransfer;
+use Generated\Shared\Transfer\PaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Generated\Shared\Transfer\SaveOrderTransfer;
+use Orm\Zed\AfterPay\Persistence\SpyPaymentAfterPay;
+use Orm\Zed\AfterPay\Persistence\SpyPaymentAfterPayOrderItem;
+use Orm\Zed\Payment\Persistence\SpySalesPayment;
+use Orm\Zed\Payment\Persistence\SpySalesPaymentMethodType;
+use Orm\Zed\Payment\Persistence\SpySalesPaymentMethodTypeQuery;
+use Spryker\Shared\Oms\OmsConstants;
+use SprykerEco\Shared\AfterPay\AfterPayConfig;
+use SprykerEco\Shared\AfterPay\AfterPayConstants;
+use SprykerEco\Zed\AfterPay\Communication\AfterPayCommunicationFactory;
+
+class AfterPayPreCheckPluginTest extends AfterPayFacadeAbstractTest
+{
+    /**
+     * @var \SprykerEcoTest\Zed\AfterPay\AfterPayZedTester
+     */
+    protected $tester;
+
+    /**
+     * @return void
+     */
+    public function testAuthorize(): void
+    {
+        // Assign
+        $activeProcessName = 'AfterPayInvoice01';
+        $this->tester->setConfig(OmsConstants::ACTIVE_PROCESSES, [$activeProcessName]);
+        $this->tester->setConfig(AfterPayConstants::AFTERPAY_AUTHORIZE_WORKFLOW, AfterPayConfig::AFTERPAY_AUTHORIZE_WORKFLOW_ONE_STEP);
+
+        $savedOrderTransfer = $this->tester->haveOrder(
+            [
+                'unitPrice' => 100,
```

Comment:
Use them as a keys of an array:
\Generated\Shared\Transfer\ItemTransfer::UNIT_PRICE
\Generated\Shared\Transfer\ItemTransfer::SUM_PRICE